core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -1025,28 +1025,17 @@ GROUP BY sal]]>
         </Resource>
         <Resource name="planBefore">
             <![CDATA[
-LogicalAggregate(group=[{0}], EXPR$1=[SUM($1)], EXPR$2=[MIN(DISTINCT $1)], EXPR$3=[SUM(DISTINCT $0)])
+LogicalAggregate(group=[{0}], EXPR$1=[SUM($1)], EXPR$2=[MIN($1)], EXPR$3=[SUM(DISTINCT $0)])
```

Comment:
Can you add another test that has multiple DISTINCTs, but can't be reduced? like `SUM(DISTINCT comm), SUM(DISTINCT sal)`?

Reply:
well, let me add a test for this.

Reply:
The intent of this test case is to test multiple distinct agg functions. Now it becomes only 1. So I recommend change  MIN(DISTINCT comm) to  SUM(DISTINCT comm) or  AVG(DISTINCT comm), so that it won't be reduced.

Reply:
> The intent of this test case is to test multiple distinct agg functions. Now it becomes only 1. So I recommend change MIN(DISTINCT comm) to SUM(DISTINCT comm) or AVG(DISTINCT comm), so that it won't be reduced.

well,  I have modified it.